src/Plugin/Validation/Constraint/InstagramEmbedCodeConstraint.php

```diff
@@ -15,7 +15,7 @@
  * @constraint(
  *   id = "InstagramEmbedCode",
  *   label = @Translation("Instagram embed code", context = "Validation"),
- *   type = { "entity", "entity_reference" }
+ *   type = { "entity", "entity_reference", "string", "string_long" }
```

Comment:
Beyond maybe changing the name of $entity to something more generic ($value?) in the validator, I don't think we do...